package.json

```diff
@@ -57,7 +61,7 @@
     "cordova-plugin-secure-storage-echo": "^5.1.1",
     "ionic-mocks": "^1.3.0",
     "lodash": "^4.17.20",
-    "moment": "2.24.0",
+    "moment": "^2.29.1",
```

Comment:
What would be the reason for this package upgrade?

Reply:
reverted back